src/main/java/com/taosdata/kafka/connect/source/SourceConfig.java

```diff
@@ -142,12 +141,12 @@ public static ConfigDef config() {
                         ConfigDef.Width.SHORT,
                         FETCH_MAX_ROWS_DISPLAY
                 )
-                .define(
-                        TABLES_CONFIG,
-                        ConfigDef.Type.LIST,
-                        Collections.EMPTY_LIST,
-                        ConfigDef.Importance.LOW,
-                        TABLES_DOC)
+//                .define(
```

Comment:
If we don't need such code, just remove them. Git log records everything, we can restore them safely if need.

Reply:
👌